src/apps/useExtensions.ts

```diff
@@ -40,35 +62,72 @@ export const mapToMenuItems = (extensions: Extension[]) =>
     onSelect: open
   }));
 
-export const useExtensions = (
-  view: AppExtensionViewEnum,
-  type: AppExtensionTypeEnum
-): Record<Target, Extension[]> => {
+export const mapToExtensionsItems = (
```

Comment:
What is _extension items_ exactly? The `mapToMenuItems` makes sense because it applied to all menus, but I don't see how this function output make it useful in places other than navigation itself. I think it makes more sense to put specific one-off mappers inside places that actually use them, rather than end up with a list of mappers in one place. If this can be useful for other scenarios, it might make sense to keep it here but I can't think of one.